javascript-bank/account.js

```diff
@@ -0,0 +1,34 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount > 0 && amount % 1 === 0) {
+    this.transactions.push(new Transaction('deposit', amount));
+    return true;
+  } else return false;
```

Comment:
I wouldn't write `return` statements like this, it's really hard to read. You should be moving this onto a new line. You want to write your code in a way that it's easily readable, and this makes it really hard for developers to skim each line and see what's happening. When using an `else` statement as well, you should be using another pair of brackets. 

Your code should look like this
```
if (amount > 0 && amount % 1 === 0) {
	this.transactions.push(new Transaction('deposit', amount));
	return true;
} else {
	return false;
}
```